src/eei.cpp

```diff
@@ -90,6 +90,78 @@ Literal EthereumInterface::callImport(Import *import, LiteralList& arguments) {
       return Literal();
     }
 
+    if (
+      import->base == Name("call") ||
+      import->base == Name("callCode") ||
+      import->base == Name("callDelegate") ||
+      import->base == Name("callStatic")
+    ) {
+      int64_t gas = arguments[0].geti64();
+      uint32_t addressOffset = arguments[1].geti32();
+      uint32_t valueOffset;
+      uint32_t dataOffset;
+      uint32_t dataLength;
+      uint32_t resultOffset;
+      uint32_t resultLength;
+
+      bool isValuePresent = import->base == Name("call") || import->base == Name("callCode");
+      evm_result call_result;
+      evm_message call_message;
+
+      if (isValuePresent) {
+      	valueOffset = arguments[2].geti32();
+	dataOffset = arguments[3].geti32();
+	dataLength = arguments[4].geti32();
+	resultOffset = arguments[5].geti32();
+	resultLength = arguments[6].geti32();
+
+	call_message.sender = msg.address;
+	call_message.value = loadUint128(valueOffset);
+	call_message.kind = (import->base == Name("callCode")) ? EVM_CALLCODE : EVM_CALL;
+	call_message.flags = 0;
+      } else {
+      	dataOffset = arguments[2].geti32();
+	dataLength = arguments[3].geti32();
+	resultOffset = arguments[4].geti32();
+	resultLength = arguments[5].geti32();
+
+	if (import->base == Name("callDelegate")) {
+	  call_message.sender = msg.sender;
+	  call_message.value = msg.value;
+	  call_message.kind = EVM_DELEGATECALL;
+	  call_message.flags = 0;
+	} else {
```

Comment:
I would got with `else if (import->base == Name("callStatic")` to be able to find it's implementation quickly.